src/paths/projects.ts

```diff
@@ -22,12 +24,23 @@ ProjectsRouter.get(
     })
 );
 
-ProjectsRouter.use('/:id', prefetch(Project));
-
-ProjectsRouter.get('/:id', read(Project));
-
-ProjectsRouter.get('/:id/metadata', (req, res) => {
-    const project = req.doc as HydratedDocument<IProject>;
+ProjectsRouter.get('/:tokenId/metadata', async (req, res) => {
```

Comment:
This endpoint makes no sense. It would have to be a different endpoint. `/projects/:id` should only be for IDs

Here are some suggestions, but you let me know which makes most sense:
- `/tokens/:tokenId`
- `/tokens/projects/:tokenId` (To account for multiple different nft types?)
- `/contracts/:tokenId`
- `/contracts/projects/:tokenId`
- `/projects/tokenId/:tokenId`
- `/projects?tokenId=<tokenId>`